resources/public/api/conf/P2.0/docs/employments-endpoint-description.md

```diff
@@ -0,0 +1,3 @@
+<p>This endpoint returns an individual's PAYE employments data for the requested time period.</p>
+<p>The fromDate and toDate are inclusive i.e. the data returned are for the time period that includes the fromDate and the toDate.</p>
+<p>Employment data is only available for tax years commencing 2013-2014.</p>
```

Comment:
Is this still true on IF? @JackBrBell - can you please confirm?

Reply:
I'll remove this for now